src/attack.rs

```diff
@@ -65,8 +65,8 @@ fn activate_hitbox(
             {
                 if let Some(fighter_data) = fighter_assets.get(fighter_meta) {
                     commands.entity(entity).insert(Collider::cuboid(
-                        fighter_data.attack.hitbox.x,
-                        fighter_data.attack.hitbox.y,
+                        fighter_data.attack.hitbox.size.x / 2.,
```

Comment:
why are you dividing the hitbox by 2 here?

Reply:
The collider expects half widths and heights like in the case of the circle it expects the radius.

Reply:
oh TRUE

Reply:
Im now noticing you updated the hitbox sizes to account for it as well so :+1: